examples_biomechanics/RebootMotionBiomechanicsSimulation_BatchInverseKineticsProcessing.ipynb

```diff
@@ -0,0 +1,1737 @@
+              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
+              "    })();\n",
+              "  </script>\n",
+              "</div>\n",
+              "    </div>\n",
+              "  </div>\n"
+            ]
+          },
+          "metadata": {},
+          "execution_count": 9
+        }
+      ]
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "if not os.path.exists('player_data'):\n",
+        "    os.mkdir('player_data')"
+      ],
+      "metadata": {
+        "id": "gB4m4dj8K75t"
+      },
+      "execution_count": 10,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "roster_result_values = {}\n",
+        "for i in roster.index:\n",
+        "    player_meta_data = PlayerMetadata(\n",
+        "        org_player_ids=[roster.loc[i,'MLBAMID'].astype(str)],\n",
+        "        session_dates=None,\n",
+        "        session_nums=None,\n",
+        "        session_date_start=datetime.date(2023, 4, 1),\n",
+        "        session_date_end=datetime.date(2023, 9, 15),\n",
+        "        year=2023,\n",
+        "        org_movement_id=None, # set the play GUID for the skeleton animation; None defaults to the first play\n",
+        "        s3_metadata=s3_metadata,\n",
+        "    )\n",
+        "\n",
+        "    primary_segment_summary_df = rtk.filter_s3_summary_df(player_meta_data, s3_df)\n",
+        "\n",
+        "    primary_segment_data_df = rtk.load_sampled_games_to_df_from_s3_paths(\n",
+        "        primary_segment_summary_df['s3_path_delivery'].tolist(), add_ik_joints=True, add_elbow_var_val=True, game_proportion=0.25, movement_proportion=0.2\n",
+        "    )\n",
+        "\n",
+        "    # Set the mass for the player in the primary segment (use kilograms)\n",
+        "    primary_player_mass = roster.loc[i,'weight']\n",
+        "\n",
+        "    # Retrieve the scaled model (both in length and mass) from AWS\n",
+        "    primary_model_xml_str = rtk.scale_human_xml(primary_segment_data_df, primary_player_mass, boto3_session=boto3_session)\n",
+        "\n",
+        "    # Simulate all the plays in the primary segment with the retrieved model\n",
+        "    primary_sim_data_df = simulate_segment_df(primary_model_xml_str, primary_segment_data_df)\n",
+        "\n",
+        "    #primary_segment_dict = rtk.load_data_into_analysis_dict(\n",
+        "    #    primary_analysis_segment, primary_sim_data_df, segment_label=primary_segment_label\n",
+        "    #)\n",
+        "\n",
+        "    # Filter dataframe\n",
+        "    filtered_df = primary_sim_data_df[(primary_sim_data_df['time_from_max_hand'] >= -0.2) & (primary_sim_data_df['time_from_max_hand'] <= 0.05)]\n",
+        "\n",
+        "    # Group by and aggregate\n",
+        "    result = filtered_df.groupby(['session_num', 'org_movement_id']).agg({\n",
+        "        'right_shoulder_rot_invdyn': ['min', 'max'],\n",
```

Comment:
probably need to make right vs left dynamic here, maybe can have it as part of the roster or pull it from the stats api

Reply:
Addressed with roster file